__tests__/groups/group.test.js

```diff
@@ -14,7 +14,7 @@ describe('Discord Groups Page', () => {
 
   beforeAll(async () => {
     browser = await puppeteer.launch({
-      headless: true,
+      headless: 'new',
```

Comment:
what is diff b/w new and true

Reply:
Chrome  plan to completely remove the old Headless from the Chrome binary and stop supporting this mode in Puppeteer later this year. 

**Difference:**
1.  Browser features and compatibility: headless: true runs the browser in a separate, alternate implementation, which means it has its own bugs and features that aren't present in headful Chrome. This can cause inconsistencies in automated browser tests that rely on having a browser extension installed or other browser-level functionality [developer.chrome.com](https://developer.chrome.com/articles/new-headless/). On the other hand, headless: 'new' unifies the Headless and headful modes, making it closer to Chrome's regular "headful" mode, and providing better compatibility with browser extensions
2. Second reason headless true is deprecating and it is recommended to use the latest headless new by chrome puppeteer team 
Source: https://developer.chrome.com/articles/new-headless/